pkg/operator/operator.go

```diff
@@ -269,31 +274,82 @@ func (o *Operator) setupAdmissionWebhooks(ctx context.Context) error {
 	}
 
 	// Keep setting the caBundle in the expected webhook configurations.
-	go func() {
-		// Only inject if we've an explicit CA bundle ourselves. Otherwise the webhook configs
-		// may already have been created with one.
-		if len(caBundle) == 0 {
-			return
-		}
-		// Initial sleep for the client to initialize before our first calls.
-		// Ideally we could explicitly wait for it.
-		time.Sleep(5 * time.Second)
+	err = ctrl.NewControllerManagedBy(o.manager).
+		Named("validating-webhook-cert-updater").
+		For(
+			&arv1.ValidatingWebhookConfiguration{},
+			builder.WithPredicates(predicate.Funcs{
+				CreateFunc: func(ev event.CreateEvent) bool {
```

Comment:
What about using the [`NewPredicateFuncs`](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/21478f108665efe083f1ae90cea8f542d3b5044c/vendor/sigs.k8s.io/controller-runtime/pkg/predicate/predicate.go#L100-L103) helper?

Reply:
We want to ignore delete events.